src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: transform 300ms;
```

you put a wrong property
```suggestion
    transition: color 300ms;
```